test/qunit/qunit.js

```diff
@@ -0,0 +1 @@
+qunit-1.20.0.js
```

Comment:
It's a symlink to the filename with version numbers. I completely forgot there are Windows users in this world. Will remove the symlinks.